src/services/rocketChat/index.js

```diff
@@ -482,6 +486,41 @@ class RocketChatChannel {
 		return Promise.resolve();
 	}
 
+	async synchronizeModerators(team) {
+		try {
+			const channel = await this.app.service('/rocketChat/channel').get(team._id);
+			const rcResponse = await request(getRequestOptions(
+				`/api/v1/groups.moderators?roomName=${channel.channelName}`,
+				{},
+				true,
+				undefined,
+				'GET',
+			));
+			let rcModerators = rcResponse.moderators;
+			const scModeratorPromises = [];
+			team.userIds.forEach(async (user) => {
+				if (this.teamModeratorRoles.includes(user.role.toString())) {
+					scModeratorPromises.push(this.app.service('rocketChat/user').get(user.userId));
+				}
+			});
+			let scModerators = await Promise.all(scModeratorPromises);
+			rcModerators = rcModerators.map(mod => mod._id);
+			scModerators = scModerators.map(mod => mod.rcId);
+			const addModerators = scModerators.filter(x => !rcModerators.includes(x));
```

Comment:
moderatorsToAdd? moderatorsToRemove?

Reply:
would be semantically wrong, as only users in only one of the sets is to be removed. These sets represent the moderators as they are currently stored in the schulcloud and rocketchat respecively, and should be named such.